src/main/app/document/DocumentManagementController.test.ts

```diff
@@ -159,6 +159,23 @@ describe('DocumentManagerController', () => {
       expect(req.locals.api.triggerEvent).not.toHaveBeenCalled();
       expect(res.json).not.toHaveBeenCalled();
     });
+
+    it('redirects if deleting & JavaScript is disabled', async () => {
```

Comment:
Should this be moved inside the `describe('Deleting files'...` block rather than being in the ` describe('Uploading files'...` block?